common/persistence/dataInterfaces.go

```diff
@@ -142,6 +142,9 @@ type (
 		CancelRequestID              string
 		StickyTaskList               string
 		StickyScheduleToStartTimeout int32
+		ClientLibraryVersion         string
```

Comment:
You made the changes to mutable state API, but where are the cassandra persistence changes to support CRUD for these new values.

Reply:
fixed